citi-onboarding/CESC-client
# (feature/footer): Created footer section

#What I did:

I created a footer section

#How to test

Open the terminal and go to this project folder

At the terminal:

git checkout development
git pull
git checkout feature / footer
git pull origin feature / footer

In VScode, open the pages and footer.

Read the changed files section on GitHub.

Test the things you think are worth testing, even the ones that aren't in this description.

Test if the page works within the expected standards
Test that elements are positioned correctly
Test if the style.css it's correct.
